message_ix_models/model/material/data_aluminum.py

```diff
@@ -524,3 +525,190 @@ def gen_mock_demand_aluminum(scenario: message_ix.Scenario) -> pd.DataFrame:
     )
 
     return demand2020_al
+
+
+def gen_data_alu_trade(scenario: message_ix.Scenario):
```

Comment:
```suggestion
def gen_data_alu_trade(scenario: message_ix.Scenario) -> Dict[str, pd.Dataframe]:
```

I'm guessing this is the return type, it might need a `from typing import Dict` to make it work. Specifying this will enable easier access of the return values in other files.